admin/tools/docker/integration-tests/localhost/run-multinode-tests.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/sh
+
+# LSST Data Management System
+# Copyright 2014 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+
+
+# Launch Qserv integration tests on one Docker host
+# using multiple Docker containers
+
+# @author  Fabrice Jammes, IN2P3/SLAC
+
+set -x
+set -e
+
+. ./env.sh
+
+
+usage() {
+  cat << EOD
+
+  Usage: $(basename "$0") [options]
+
+  Available options:
+    -h          this message
+    -C          Rebuild the images from scratch
+
+  Launch Qserv integration tests on one Docker host 
+
+EOD
+}
+
+# get the options
+while getopts hC c ; do
+    case $c in
+            h) usage ; exit 0 ;;
+            C) CACHE_OPT="--no-cache=true" ;;
+            \?) usage ; exit 2 ;;
+    esac
+done
+shift "$((OPTIND-1))"
+
+if [ $# -ne 0 ] ; then
+    usage
+    exit 2
+fi
+
+# Start dynamic DNS for local Docker containers 
+docker rm -f docker_spy || echo "No existing container for docker_spy"
+docker run --name docker_spy --detach=true -e "DNS_DOMAIN=$DNS_DOMAIN" -v /var/run/docker.sock:/var/run/docker.sock iverberk/docker-spy
```

Comment:
Passing the docker sock down into a container is a fairly risky thing to do.  I think this warrants a comment noting that this is a security concern.